executor/api/grpc/seldon/server.go

```diff
@@ -0,0 +1,52 @@
+package seldon
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	"github.com/seldonio/seldon-core/executor/api/client"
+	"github.com/seldonio/seldon-core/executor/api/grpc"
+	"github.com/seldonio/seldon-core/executor/api/grpc/seldon/proto"
+	"github.com/seldonio/seldon-core/executor/api/payload"
+	"github.com/seldonio/seldon-core/executor/predictor"
+	"github.com/seldonio/seldon-core/operator/apis/machinelearning/v1"
+	"net/url"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+type GrpcSeldonServer struct {
+	Client    client.SeldonApiClient
+	predictor *v1.PredictorSpec
+	Log       logr.Logger
+	ServerUrl *url.URL
+	Namespace string
+}
+
+func NewGrpcSeldonServer(predictor *v1.PredictorSpec, client client.SeldonApiClient, serverUrl *url.URL, namespace string) *GrpcSeldonServer {
+	return &GrpcSeldonServer{
+		Client:    client,
+		predictor: predictor,
+		Log:       logf.Log.WithName("SeldonGrpcApi"),
+		ServerUrl: serverUrl,
+		Namespace: namespace,
+	}
+}
+
+func (g GrpcSeldonServer) Predict(ctx context.Context, req *proto.SeldonMessage) (*proto.SeldonMessage, error) {
```

Comment:
As `GrpcSeldonServer` struct is non-primitive, should the method receiver be a pointer?